### ISSUE TITLE
Is it possible to provide symbolic observation for each agent?

### ISSUE THREAD

**Issue by nikepupu** [allenai/ai2thor#644]
Right now the global observation (e.g. visibility or isObstrated etc is global [shared between agents] not for each individual agent). Is it possible to provide symbolic observation for each agent?

**Comment by mattdeitke** [comment]
Yes, it is possible to get an `Event` for each agent in a multi-agent setting.

Here is an incomplete multi-agent tutorial that I've put together with Colab which currently addresses how to access frames, each agent's `Event` from a `MutliAgentEvent`, and how to take actions:
https://colab.research.google.com/drive/1ityVEzHNN7k4Xek7-yfX_x_VRWNtO2wz?usp=sharing

I'm planning on adding a bunch of these to the website and documentation soon. The documentation functionality regarding multi-agent systems is currently pretty sparse. Also, let me know if you have any suggestions!
